build/commands/lib/config.js

```diff
@@ -324,7 +324,7 @@ Config.prototype.buildArgs = function () {
     ...this.extraGnArgs,
   }
 
-  if (!this.isOfficialBuild()) {
+  if (!args.is_official_build) {
```

Comment:
these two things should always be the same or it's going to create confusion like this

Reply:
this is wrong
```is_official_build: this.isOfficialBuild() && !this.isAsan(),```

Reply:
`this.isOfficialBuild()` should include `!this.asAsan()`

Reply:
actually to avoid changing other logic lets change isOfficialBuild -> isReleaseBuild so there's no confusing between this.isOfficialBuild and args.is_official_build